README.md

```diff
@@ -44,12 +44,12 @@ Explore our [demo app](https://demo.airbyte.io/).
 
 ### Run Airbyte locally
 
-You can run Airbyte locally with Docker.
+You can run Airbyte locally with Docker. The shell script below will retrieve the requisite docker files from the [platform repository](https://github.com/airbytehq/airbyte-platform) and run docker compose for you.
 
 ```bash
 git clone --depth 1 https://github.com/airbytehq/airbyte.git
 cd airbyte
-docker compose up
+./run-ab-platform.sh
```

Comment:
Do we need to update the [deployment guides](https://docs.airbyte.com/category/deploy-airbyte-open-source) with the new command?

Reply:
Local needs to be updated with the shell script here. The other guides were updated in https://github.com/airbytehq/airbyte/pull/23240

Reply:
I pushed an update to local deployment in this pull request: https://github.com/airbytehq/airbyte/pull/22947